pkg/metrics/constants.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"go.opencensus.io/tag"
+	"knative.dev/pkg/metrics/metricskey"
+)
+
+type PodName string
+type ContainerName string
+
+var (
+	// Create the tag keys that will be used to add tags to our measurements.
+	// Tag keys must conform to the restrictions described in
+	// go.opencensus.io/tag/validate.go. Currently those restrictions are:
+	// - length between 1 and 255 inclusive
+	// - characters are printable US-ASCII
+	NamespaceNameKey = tag.MustNewKey(metricskey.LabelNamespaceName)
+
+	BrokerNameKey        = tag.MustNewKey(metricskey.LabelBrokerName)
+	EventTypeKey         = tag.MustNewKey(metricskey.LabelEventType)
+	TriggerNameKey       = tag.MustNewKey(metricskey.LabelTriggerName)
+	TriggerFilterTypeKey = tag.MustNewKey(metricskey.LabelFilterType)
+
+	ResponseCodeKey      = tag.MustNewKey(metricskey.LabelResponseCode)
+	ResponseCodeClassKey = tag.MustNewKey(metricskey.LabelResponseCodeClass)
+
+	PodNameKey       = tag.MustNewKey(metricskey.PodName)
+	ContainerNameKey = tag.MustNewKey(metricskey.ContainerName)
+)
```

Comment:
Maybe name this tags.go?

Reply:
@ian-mi I'm not sure why `e2e-secret-tests.sh` is showing up in my PR, I sync'd with master and I guess it was caught in the middle for some reason but I made no changes and the permissions are in line with https://github.com/google/knative-gcp/commit/12200f7088cd728b7e1a683fe5b6db80b7a36468#diff-098f6bcd4621d373cade4e832627b4f6

Reply:
Renamed to `tags.go`